pkg/kubelet/root_context_linux.go

```diff
@@ -33,3 +33,7 @@ func (kl *Kubelet) getRootDirContext() (string, error) {
 	// Get the SELinux context of the rootDir.
 	return selinux.Getfilecon(kl.getRootDir())
 }
+
```

Comment:
> we will probably wind up renaming this file to something like selinux_linux.go shortly.

+1

> I'm still hung up on rejecting a pod that specifies SELinux options when the cluster is running in a non-SELinux enabled configuration. I don't see a reason it should be an fatal condition, anything that would normally be setting labels would just ignore them based on the cluster settings anyway, I assume.

So lets say I have an untrusted pod. I am worried about running it so I run it under the label
'untrusted_u:untrusted_r:untrusted_t' to which my policy give very limited access. In that case if a the above pod is allowed to run on a system with SELinux not enabled then it goes against what the user wanted